src/content/admin/dashboard.mdx

```diff
@@ -150,7 +150,30 @@ This table provides a consolidated view about Preview Environments, including:
 - Last seen
 
 ## Command Line (CLI)
-To add docs in separate PR
+
+The Command Line section allows you to manage how developers interact with Okteto via their CLI.
+Here, you can enable [remote execution by default](core/remote-execution.mdx) and configure the minimum supported CLI version that developers can use.
+
+As an Okteto Admin, you can now enforce a minimum supported CLI version for your developers.
+This feature ensures compatibility, encourages feature adoption, and maintains a consistent experience across your team.
+
+### How Enforcing a Minimum Supported CLI Version Works
+
+**Admin Configuration**
+
+- Set the minimum supported CLI version in the Okteto Admin Dashboard with these options:
+  - **Any Version**: No restriction, developers can use any CLI version
+  - **Current Version (e.g., 3.2.x)**: Restricts developers to the latest CLI version packaged with the cluster
+  - **Previous Version (e.g., 3.1.x)**: Supports the latest version and the immediately prior minor version
+  - **Two Previous Versions (e.g., 3.0.x - 3.2.x)**: Supports the latest version and the previous two minor versions
+
+**CLI Version Enforcement**
+
+- Commands fail if a developer uses a CLI version outside the allowed range, displaying an error prompting them to update their CLI.
```

Comment:
It doesn't fail for the range, newer versions are always supported

Reply:
Also, I think we should mention here that this starts to apply from `3.2` and above. I mean, `3.1` doesn't have the code validating the backend version, so even if the backend is forcing to use only `3.2`, users might still use `3.1` CLI